src/qt/forms/debugwindow.ui

```diff
@@ -1592,6 +1592,84 @@
                 </widget>
                </item>
                <item row="23" column="0">
+                <widget class="QLabel" name="peerAddrRelayEnabledLabel">
+                 <property name="toolTip">
+                  <string>Whether we relay addresses to this peer.</string>
```

Comment:
Could you add an `extracomment` attribute as a translator comment for newly added tooltip strings?

As an example see dbde0558ce73db4c901dbaa2eddc634701fa1d0d.

Reply:
Thanks -- done.